src/Frame.cc

```diff
@@ -24,8 +24,8 @@ Frame::Frame(const Point &LowerLeft, const Point &UpperRight)
 }
 
 
-Frame::Frame(const double &xmin, const double &ymin,
-	     const double &xmax, const double &ymax)
+Frame::Frame(double xmin, double ymin,
```

Comment:
As commented elsewhere, could you replace this with the version of Frame that takes (Point(xmin, xmax), Point(ymin, ymax)) instead? This format seems prone to user error.